cadasta/gui/tools/wizard/login_wizard.py

```diff
@@ -78,7 +78,7 @@ def __init__(self, parent=None, iface=None):
         self.set_logo()
 
     def set_logo(self):
-        filename = resources_path("images/white_icon.png")
+        filename = resources_path('images/white_icon.png')
```

Comment:
Is it not better `resources_path('images', 'white_icon.pmg')` ? The function `resources_path` will handle the OS separator.